Dockerfile

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 FROM ubuntu:xenial
 
-RUN apt-get update && apt-get install -y python3-pip curl
-RUN pip3 install tensorboard
+RUN apt-get update && apt-get install -y python-pip curl
+RUN pip install --upgrade tensorflow tensorboard
```

Comment:
I don't think we actually need tensorflow here. Is there a reason the tensorboard standalone package is not sufficient? It significantly decreases image size.

Reply:
I had to add it as I was getting an error when initializing tensorboard re 'tensorflow missing'. My guess is that is still partially use to read the checkpoint files.

Reply:
Oh wow. Yeah that's really bad. Can you file an issue on the Tensorboard repository. pip packages should install all their dependencies.